pkgs/applications/networking/cluster/kubectl-node-shell/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "kubectl-node-shell";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "kvaps";
+    repo = "kubectl-node-shell";
+    rev = "v${version}";
+    sha256 = "sha256-dAsNgvHgquXdb2HhLDYLk9IALneKkOxQxKb7BD90+1E=";
+  };
+
+  installPhase = ''
+    install -m755 ./kubectl-node_shell -D $out/bin/kubectl-node_shell
```

Comment:
Missing `runHook preInstall` and `runHook postInstall`

Reply:
done